pkg/logs/processor/processor.go

```diff
@@ -52,92 +55,74 @@ func (p *Processor) run() {
 	for msg := range p.inputChan {
 		shouldProcess, redactedMessage := p.applyRedactingRules(msg)
 		if shouldProcess {
-			extraContent := p.computeExtraContent(msg)
-			apikeyString := p.computeAPIKeyString(msg)
-			payload := p.buildPayload(apikeyString, redactedMessage, extraContent)
-			msg.SetContent(payload)
-			p.outputChan <- msg
+			content, err := p.computeContent(msg, redactedMessage)
+			if err == nil {
```

Comment:
To be more go friendly you could use the pattern:
```
if err != nil {
   ...
   continue
}
```

Reply:
👍